Lib/urllib/request.py

```diff
@@ -1136,15 +1136,17 @@ def get_authorization(self, req, chal):
         A2 = "%s:%s" % (req.get_method(),
                         # XXX selector: what about proxies and full urls
                         req.selector)
-        if qop == 'auth':
+        # NOTE: As per  RFC 2617, when server sends "auth,auth-int", the client could use either `auth`
+        #     or `auth-int` to the response back. we use `auth` to send the response back. 
```

Comment:
@PypeBros you have a trailing space here:

```suggestion
        #     or `auth-int` to the response back. we use `auth` to send the response back.
```